### ISSUE TITLE
supernova reboot not working properly

### ISSUE THREAD

**Issue by adcxyz** [supercollider/supercollider#3794]
When rebooting a running supernova server, it does not register correctly for notification, 
and thus never reaches serverRunning true. 
The same code works with scsynth, so it could be a timing issue in the lang-server messaging.
tested on 2015 macbookpro, SC393, develop branch.

```supercollider
Server.killAll;

 // switch between scsynth and supernova 
Server.scsynth;   

// with scsynth, this works
// reboot with server off works
s.reboot({ 
	"has quit - rebooting...".postln;
	s.doWhenBooted({ "... and rebooted ok.".postln });
});

// and reboot with running server works
s.reboot({ 
	"has quit - rebooting...".postln;
	s.doWhenBooted({ "... and rebooted ok.".postln });
});

// 
Server.supernova;  
s.quit;
// reboot with server off works
s.reboot({ 
	"has quit - rebooting...".postln;
	s.doWhenBooted({ "... and rebooted ok.".postln });
});

// and reboot with running supernova server fails,
// reboot does not reach serverRunning == true: 
s.reboot({ 
	"has quit - rebooting...".postln;
	s.doWhenBooted({ "... and rebooted ok.".postln });
});

// it does not request notify and clientID, so 
s.serverRunning // false

// calling startAliveThread separately fixes notification:
s.statusWatcher.startAliveThread;
s.serverRunning // true
```

**Comment by adcxyz** [comment]
Found it: supernova takes a bit longer to really quit after sending the /quit status message. 
so watchQuit should wait for the process to actually have quit ... 
will prepare a PR for it.

**Comment by adcxyz** [comment]
closed with #3848
